packages/apktool.vm/tools/chocolateyinstall.ps1

```diff
@@ -4,18 +4,24 @@ Import-Module vm.common -Force -DisableNameChecking
 try {
   $toolName = 'apktool'
   $category = 'Java & Android'
-  $shimPath = 'bin\apktool.exe'
+  $rawToolPath = Join-Path ${Env:RAW_TOOLS_DIR} "$toolName" -Resolve
```

Comment:
```suggestion
  $rawToolPath = Join-Path ${Env:RAW_TOOLS_DIR} "$toolName"
```

Sorry, we need to remove the `-Resolve` as they are before the path is created and add a `VM-Assert-Path` after the `Get-ChocolateyWebFile`